.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
             matrix:
                 php: [7.4, 8.0]
                 symfony: [^4.4, ^5.2]
-                sylius: [~1.8.0, ~1.10.0]
+                sylius: [~1.8.0, ~1.10.0, ~1.11.0]
```

Comment:
Can you add Sylius `1.9` here? I know it wasn't present before but It'd be nice to have it here as we're supporting it in our `composer.json`. Probably there may be a need to exclude some combination in `exclude` section below, check in another plugin :P.

Reply:
Done, Sylius 1.9 require additional package